secure_redis/decorators.py

```diff
@@ -0,0 +1,159 @@
+from __future__ import (absolute_import, division, print_function,
```

Comment:
Actually these are copied from `rq.decorators`. I'm planning to remove `absolute_import` and `print_function` since these don't seem to be used. But I'd like to keep `division` since I'm worrying about hidden problem behind this.